docs/GLOSSARY.md

```diff
@@ -0,0 +1,19 @@
+# Terminology
+
+Defined terms in the documentation for Project Factory are capitalized and have
+specific meaning within the domain of knowledge
```

Comment:
Nitpick - let's add a period to this sentence.

Reply:
Good catch, thanks!  Added `.`